src/routes/manage-selections/ManageSelectionsScreen.tsx

```diff
@@ -0,0 +1,112 @@
+
+const Root = styled.ScrollView`
+  display: flex;
+  background-color: ${props => props.theme.colors.background};
+  padding: ${props => props.theme.spacings.md};
+`
+
+const SectionHeading = styled(Subheading)`
+  padding-top: ${props => props.theme.spacings.xl};
+`
+
+const Padding = styled.View`
+  padding-bottom: ${props => props.theme.spacings.xxl};
+`
+
+interface Props {
+  navigation: StackNavigationProp<RoutesParams, 'ManageDisciplines'>
+}
+
+const ManageSelectionsScreen = ({ navigation }: Props): ReactElement => {
+  const {
+    data: selectedProfessions,
+    loading,
+    error,
+    refresh: refreshSelectedProfessions
+  } = useReadSelectedProfessions()
+  const { data: customDisciplines, refresh: refreshCustomDisciplines } = useReadCustomDisciplines()
+
+  const refresh = useCallback(() => {
+    refreshCustomDisciplines()
+    refreshSelectedProfessions()
+  }, [refreshCustomDisciplines, refreshSelectedProfessions])
+
+  useFocusEffect(refresh)
+
+  const Item = ({ item }: { item: Discipline }): JSX.Element => {
+    const unselectProfessionAndRefresh = (item: Discipline) => {
+      removeSelectedProfession(item).then(refreshSelectedProfessions).catch(reportError)
+    }
+    return (
+      <ListItem
+        icon={item.icon}
+        title={item.title}
+        rightChildren={
+          <Pressable onPress={() => unselectProfessionAndRefresh(item)} testID='delete-icon'>
+            <CloseIconRed />
+          </Pressable>
+        }
+      />
+    )
+  }
+
+  const navigateToProfessionSelection = () => {
+    navigation.navigate('ScopeSelection', { initialSelection: false })
+  }
+
+  const navigateToAddCustomDiscipline = () => {
+    navigation.navigate('AddCustomDiscipline')
+  }
+
+  return (
+    <Root contentContainerStyle={{ flexGrow: 1 }}>
+      <Heading>{labels.manageDisciplines.heading}</Heading>
+      <SectionHeading>{labels.manageDisciplines.yourProfessions}</SectionHeading>
+      <HorizontalLine />
+      <ServerResponseHandler error={error} loading={loading} refresh={refreshSelectedProfessions}>
+        {selectedProfessions?.map(profession => (
+          <Item key={profession.id} item={profession} />
+        ))}
+      </ServerResponseHandler>
+      <AddElement onPress={navigateToProfessionSelection} label={labels.manageDisciplines.addProfession} />
+
+      <SectionHeading>{labels.manageDisciplines.yourCustomDisciplines}</SectionHeading>
+      <HorizontalLine />
+      <ServerResponseHandler error={error} loading={loading} refresh={refreshCustomDisciplines}>
+        {customDisciplines?.map(customDiscipline => (
+          <CustomDisciplineItem key={customDiscipline} refresh={refreshCustomDisciplines} apiKey={customDiscipline} />
+        ))}
+      </ServerResponseHandler>
+
+      <AddElement
+        onPress={navigateToAddCustomDiscipline}
+        label={labels.home.addCustomDiscipline}
+        explanation={labels.manageDisciplines.descriptionAddCustomDiscipline}
+      />
+      <Padding />
```

Comment:
can't find any difference without this view container. whats the purpose?

Reply:
If you add 5 professions the last element is close to the bottom.